pf/internal/convert/convert_test.go

```diff
@@ -27,17 +28,28 @@ import (
 )
 
 type convertTurnaroundTestCase struct {
-	name    string
-	ty      tftypes.Type
-	val     tftypes.Value
-	prop    resource.PropertyValue
-	normVal func(tftypes.Value) interface{}
+	name     string
+	ty       tftypes.Type
+	val      tftypes.Value
+	prop     resource.PropertyValue
+	normVal  func(tftypes.Value) any
+	normProp func(resource.PropertyValue) any
 }
 
 func TestConvertTurnaround(t *testing.T) {
 	t.Parallel()
 
 	cases := convertTurnaroundTestCases(tftypes.String, resource.NewStringProperty, "", "test-string")
+	cases = append(cases, convertTurnaroundTestCases(tftypes.String, func(x string) resource.PropertyValue {
```

Comment:
Nit: since these don't really turnaround, we could have just added parsing test cases rather than extending the turnaround test setup. But this works.

Reply:
I wanted us to confirm that we turn around the correct "value", even if the types don't line up.